salt/config/__init__.py

```diff
@@ -1397,6 +1398,7 @@ def _gather_buffer_space():
     'update_url': False,
     'update_restart_services': [],
     'retry_dns': 30,
```

Comment:
Seems like the default should be 0 since the type is int

Reply:
Updated the default to include None, if the value is 0 then it will always count down and I wanted the default to remain the same, trying indefinitely.